src/Controller/GeocoderController.php

```diff
@@ -1,8 +1,4 @@
 <?php
-/**
- * @file
- * Contains \Drupal\geocoder_autocomplete\Controller\GeocoderController.
- */
```

Comment:
tnx. Let me check ... once done, I will merge your pull request. Tnx so much for all this ...